projects/plugins/jetpack/extensions/blocks/blogroll/components/blogroll-appender-results/index.js

```diff
@@ -12,7 +13,12 @@ export default function BlogrollAppenderResults( { results, showPlaceholder, onS
 			{ results.length > 0 && (
 				<ul aria-live="polite">
 					{ results.map( result => (
-						<li key={ result.blog_id }>
+						<li
+							key={ result.blog_id }
+							className={ classNames( 'jetpack-blogroll__appender-result-container', {
+								'is-disabled-result': result?.duplicateRecommendation,
+							} ) }
+						>
```

Comment:
Let's change the `a` below into a button and disable it. Items that have `pointer-events: none` are still keyboard navigable and the site will be added when people tab-nav and press enter.